packages/communication-react/review/communication-react.api.md

```diff
@@ -1932,6 +1934,7 @@ export type MessageThreadProps = {
     onSendMessage?: (messageId: string) => Promise<void>;
     disableEditing?: boolean;
     strings?: Partial<MessageThreadStrings>;
+    fileDownloadHandler?: FileDownloadHandler;
```

Comment:
dont hate me but.. what if this was also on the ChatMessage interface?
(thinking of the idea of if a chat message could exist outside of the message thread)

Reply:
If contoso is willing to provide a different `fileDownloadHandler` function per message.
I am sure that logic can be wrapped inside a single `fileDownloadHandler` too. :P

Reply:
Ignore this actually. You have already added this to the ChatMessageAsAWhatever which makes sense